### PR TITLE
Drop page support

### PR DESCRIPTION
**Pull request by ildyria**
I took the difficult decision to drop the page support.
This functionality is not visible to the normal admin and one needs to be very familiar with the code to use it.
Maybe such can be added later but for now it is not a necessary feature of Lychee.

### CODE REVIEW

**Review by qwerty287 — APPROVED** [review]
Looks good :+1:

File: resources/views/landing.blade.php [hunk 1]
```diff
@@ -18,7 +18,15 @@
     </div><!-- header -->
 
 
-    @include('includes.menu')
```
Comment (nagmat84) [inline resources/views/landing.blade.php:21]: If the include is removed, the file `resources/view/includes/menu.blade.php` can be deleted as well.

**Review by nagmat84 — CHANGES_REQUESTED** [review]

File: resources/views/landing.blade.php (hunk 1 above)
Comment (ildyria) [reply resources/views/landing.blade.php:21]: fixed with [0fe09e8](https://github.com/LycheeOrg/Lychee/pull/1489/commits/0fe09e8dd5763b1aa49aeb017d00f1ba09e8d8ad)

**Review by nagmat84 — APPROVED** [review]

**Review by qwerty287 — APPROVED** [review]
This is ready, or why don't we merge it?

**Comment by ildyria** [comment]
First merge the install fix, release v4.6.0 and then merge all the waiting 4.6.1 pr. 🙂

⁣Télécharger BlueMail pour Android ​

Le 4 sept. 2022, 14:36, à 14:36, qwerty287 ***@***.***> a écrit:
***@***.*** approved this pull request.

**Comment by qwerty287** [comment]
Yeah, I think I understood (unfortunately *after* I merged https://github.com/LycheeOrg/Lychee/pull/1490). 😬

**Pull request merged**